core/kotlinx-coroutines-core/test/internal/LockFreeTaskQueueLinearizabilityTest.kt

```diff
@@ -14,12 +14,13 @@ import kotlin.test.*
 
 @OpGroupConfigs(OpGroupConfig(name = "consumer", nonParallel = true))
 @Param(name = "value", gen = IntGen::class, conf = "1:3")
-class LockFreeMPSCQueueLinearizabilityTest : TestBase() {
-    private lateinit var q: LockFreeMPSCQueue<Int>
+class LockFreeTaskQueueLinearizabilityTest : TestBase() {
```

Comment:
While changes in the queue look correct (I will take one more look in the morning), I'd like to see more robust linearizability tests as soon as #783 will be ready

Reply:
Makes sense